rustls/src/server/tls12.rs

```diff
@@ -968,6 +976,8 @@ impl State<ServerConnectionData> for ExpectFinished {
 
 // --- Process traffic ---
 struct ExpectTraffic {
+    // only available if `config.enable_secret_extraction` is true
```

Comment:
if you agree with my feedback about saying "Some" instead of "available" it applies here as well.